# Gives a list of unique weeds
UniqueWeed[] weeds